index.html

```diff
@@ -0,0 +1,552 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Pong Landing Page</title>
+
+  <!--Bootstrap CDN-->
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous" />
+  <link rel="stylesheet" type="text/css" href="/static/assets/landing.css" />
+  <!--Google Fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Bungee+Shade|Press+Start+2P&display=swap" rel="stylesheet">
+  <!--Font Awesome-->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css">
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.20/css/dataTables.bootstrap4.min.css">
+</head>
+
+<body>
+  <!--Title-->
+  <header>
+    <div class="mx-auto" style="width: 900px;">
+      <h1>X-PONG</h1>
+      <h2>Play <span>extreme</span> pong with your friends online!
+        <!-- Create New Game Room Button trigger modal -->
+        <button id="create-rm-btn" type="button" class="btn btn-primary" data-toggle="modal"
+          data-target="#exampleModalCenter">
+          Create New Room +</i>
+        </button>
+      </h2>
+    </div>
+  </header>
+
+
+  <!--New Game Room Modal -->
+  <div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle"
+    aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="exampleModalCenterTitle" style="color:black">New Room
+            Details</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <form id="game-form">
+            <div class="form-group" id="new-rm-name">
+              <label for="exampleFormControlInput1" style="color: black;">Room Name</label>
+              <input type="email" class="form-control" id="exampleFormControlInput1"
+                placeholder="your awesome room name">
+            </div>
+            <div class="form-group">
+              <label for="exampleFormControlInput1" style="color: black;">Created by</label>
+              <input type="email" class="form-control" id="exampleFormControlInput1" placeholder="your name">
+            </div>
+            <div class="form-group">
+              <label for="exampleFormControlTextarea1" style="color: black;">Description</label>
```

Comment:
First off, the modal looks great!  I don't believe we're going to need a field "description" though.  Players can expected that each room will be a game of X-pong.  For the purposes of this placeholder, I think it'll be best to have only fields we _know_ will be utilized immediately.

Reply:
Thanks! That makes sense, appreciate the feedback!